core/sql/common/json.h

```diff
@@ -0,0 +1,124 @@
+#ifndef JSON_H
```

Comment:
Yes, and also please try to follow the naming conventions for source file names. For example, this file in the common directory should probably be called ComJSON.h or similar. That should make it much easier to find. Otherwise, if someone would come across a name like json.cpp or json.h, they would assume that it is not part of Trafodion and/or they would have to do quite a bit of searching to figure out where the file comes from.